src/js/modules/ListLink.js

```diff
@@ -6,12 +6,11 @@ class ListLink extends Component {
 
   render() {
 
-    const { id, url, text, isBoxType } = this.props;
+    const { url, text, linkStyle } = this.props;
 
     return (
       <Link
-        className={ `coa-ListLink ${(isBoxType && "coa-ListLink--boxprimary" )}` }
-        key={id}
+        className={ `coa-ListLink coa-ListLink--${(linkStyle)}` }
```

Comment:
I like how you pass the modifier into classname right from the props. This makes me think that we should get back to adding PropTypes in our components. It will help us avoid a situation where this modifier gets an unexpected value. Should I add that to the backlog?

Reply:
yes plz.